init.lua

```diff
@@ -25,9 +25,13 @@
 
 ]]--
 
-filter = { registered_on_violations = {} }
+filter = {
+	registered_on_violations = {},
+	registered_filters = {}
+}
 local words = {}
 local muted = {}
+local ret_val = {}
```

Comment:
Shouldn't be declared in this context. Don't reuse temporary variables like `ret_val`, it's a bad habit and may lead to difficult to solve bugs.

Reply:
Understood. Should it be declared in `check_message` itself instead?

Reply:
nvm; done